src/wmnds/patterns/footer/_footer.scss

```diff
@@ -21,4 +21,169 @@
       display: block;
     }
   }
+
+  // footer v2 only styles
+  &--v2 {
+    padding: $size-lg $size-xs;
+
+    .wmnds-btn {
+      display: block;
+      color: get-color(primary);
+      background-color: $white;
+    }
+
+    .wmnds-footer__link {
+      color: currentColor;
+      font-weight: normal;
+      text-decoration: none;
+
+      &:hover {
+        color: currentColor;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-md - 1) {
+    .wmnds-footer__updates-col {
+      padding-bottom: $size-md * 1.25;
```

Comment:
Update this so it uses one of the sizes....
We shouldn't multiply or manipulate the sizes as this ensures correct spacing throughout.